packages/server/src/index.ts

```diff
@@ -144,6 +156,29 @@ app.post("/login", vValidator("json", loginUserSchema), async (c) => {
 	}
 });
 
+app.get("/auth/me", authMiddleware, async (c) => {
+	const db = drizzle(c.env.DB);
+
+	const userId = c.get("jwtPayload").sub as string;
```

Comment:
これas 使わずに書けない？asは型安全じゃないからよっぽどのことがない限り使わないかな

Reply:
ついでに直接受取るんじゃなくて
```suggestion
	const payload = c.get("jwtPayload");
	console.log("userId", payload.sub)
```
みたいなほうが見やすい？かも。（俺の好み）